code/index.html

```diff
@@ -2,21 +2,121 @@
+        
+      <h1>birdviews</h1>
+       <div class="navigation"> 
+        <a class="link1" href="https://www.klart.se/" target="_blank">Weather</a>
+        <a class="link2" href="https://birdlife.se/">Birdlife</a>
+        <a class="link3" href="https://twitter.com/?lang=sv">Social</a>
+        <a class="link4" href="https://www.aftonbladet.se/">Hooman world</a>
+      </div>
+    </section>
     </header>
 
-    <section class="big-news">
+    <section class="site-name">
+      <h2>birdviews</h2>
+      <p class="slogan"><i>-your weekly guide to life with wings!</i></p>
+   </section>
+    
       <!-- The big news content -->
+    <section class="big-news">
+        <img class="big-news-image" src="https://windmillskill.com/sites/windmillskill/files/styles/hosted_core_wide/public/grulla-montes-cierzo.jpg?itok=x8VbCXV5" alt="dead bird"/>
+        <p class="big-news-text">
+          <b>Wind Farm in Stockholm</b> <br>
+          New wind farm is planned to be set up just outside of Stockholm. We urge everyone to keep up with the construction and find new way to fly when flying through the south of the city. Our reporters haven’t yet found out the exact date for building to commence. As soon as we have the information we will post it.  
+        </p>
     </section>
-
-    <section class="">
+      
       <!-- The other news  -->
-    </section>
-  </body>
+    <section class="other-news">
+    <div class="card-pic">
+      <p>
+        <img class="world-map" src="https://www.birdlife.org/sites/default/files/styles/full_1140x550/public/flyways_infographic_thin_1.jpg?itok=dgXS3Qja" alt="World map"/>
+        <div class="container-text">
+        <a class="other-news-link" href="https://www.birdlife.org/worldwide/programmes/migratory-birds">Flying south?</a><br>Winter is coming.
+      </p></div>
+    </div>
+
+    <div class="card-pic">
+      <p>
+      <img class="winter-birds" src="https://www.blomsterlandet.se/b1dd0b38/contentassets/331c691400904043bc1c5c2a2bc2824b/23436534-blames-och-en-talgoxe-vid-en-fagelmatning.jpg" alt="Winter birds"/>
+      <div class="container-text">
+      Staying for christmas?<br>
+      <a class="other-news-link" href="https://www.healthline.com/nutrition/bulking">How to bulk up.</a>  
+    </p></div>
+    </div>
+
+    <div class="card">
+      <p>
+        <b>Short News</b> <br>
+        Hawk assaulted by gang of sparrows, needed to seek medical attention. 
+        Seagull association try to take over fish ‘n chip company. 
+        150 new reports of birds crashing into windows (1 553 013 this year). 
+        0 new reports of the bird flue, passing it on to hoomans seemed to work.
+      </p>
+    </div>
+
+      <div class="card-pic">
+      <p>
+      <img class="owl" src="https://pbs.twimg.com/media/EWhnmsJWoAAV_RN.jpg" alt="owl"/><br>
+      <div class="container-text">New winner of the "Long leg contest" <b><i>miss Henrietta Owl-Smith.</i></b> 
+      </p></div>
```

Comment:
p-tag does not have to be wrapped around the img since they are all in a div which contains them. Now the img becomes the child of the p-tag, this is ofc fine but maybe you want to be able to style them individually, in that case I think you have more freedom if you keep them as siblings in the div-tag.